wme_junctionangle.user.js

```diff
@@ -1919,7 +1968,7 @@ function run_ja() {
 					keepInstructionColor: "\"Pysy vasemmalla/oikealla\"-ohjeen väri",
 					exitInstructionColor: "\"Poistu\"-ohjeen väri",
 					turnInstructionColor: "\"Käänny\"-ohjeen väri",
-					problemColor: "Vältettävien kulmien väri",
+					problemColor: "Väri laitonta sotaharjoitukset",

```

Comment:
\o/
This now translates as "Color illegals military excercises". Gotta love automatic translations =D